integration-tests/ledger-signer-failing-tests.spec.ts

```diff
@@ -20,8 +20,8 @@ CONFIGS().forEach(({ lib, setup }) => {
             done();
         });
 
-        describe('Tests where user declines all prompts on the ledger device', () => {
-            it('Should throw error when user declines to provide public key', async (done) => {
+        describe('Test to verify that user can decline all prompts on the ledger device', () => {
+            it('As a User I want to verify that Ledger will throw an error when I declined to provide public key', async (done) => {
```

Comment:
Should we use the same format as the other files "Verify that..." without "As a User I want to"?

Reply:
Changed to:
'Verify that Ledger will throw an error when no public key is provided'